javascript/src/main/resources/javascript/ManifestationInfo.use.js

```diff
@@ -63,27 +63,34 @@ var ManifestationInfo = (function() {
 
 
     /**
-     * Function that extracts title from the provided DC stream
+     * Function that extracts title from either the local data stream
+     * or if not present there the common data stream
      *
      * @type {function}
      * @syntax ManifestationInfo.getTitle( dcStreamXml )
-     * @param {Document} dcStreamXml the dc stream as xml
+     * @param {Document} commonData the common stream as xml
+     * @param {Document} localData the local stream as xml
      * @return {String} the extracted title
      * @function
      * @name ManifestationInfo.getTitle
      */
 
-    function getTitle( dcStreamXml ) {
+    function getTitle( commonData, localData ) {
 
         Log.trace( "Entering: ManifestationInfo.getTitle function" );
 
-        var title = XPath.selectText( "/oai_dc:dc/dc:title", dcStreamXml );
+        //first check if localData has a full title
+        var title = XPath.selectText( '/ting:localData/dkabm:record/dc:title[not(@xsi:type="dkdcplus:full")]', localData );
```

Comment:
kommentaren passer ikke med det der gøres (fordi man netop leder efter "not full")
så i stedet skulle der nok stå 
//first check if localData has a dc:title (not full title)